engine/src/main/java/org/entando/entando/web/common/model/PagedMetadata.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2015-Present Entando Inc. (http://www.entando.com) All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+package org.entando.entando.web.common.model;
+
+import java.util.List;
+
+import com.agiletec.aps.system.common.model.dao.SearcherDaoPaginatedResult;
+
+public class PagedMetadata<T> {
+
+    private int page;
+    private int size;
+    private int last;
+    private int count;
+
+    //@JsonIgnore
```

Comment:
Remove

Reply:
thanks for pointing me out. This annotation must be active